modules/services/queen_bee_svc/include/queen_bee_svc.h

```diff
@@ -38,6 +38,7 @@ class QueenBeeSvc : public kpsr::Service {
   void processImg(const kpsr::vision_ocv::ImageData &img);
 
   std::vector<cv::Vec3f> _circles;
+  bool _receivedImage;
```

Comment:
what is the use of ```_receivedImage```?

Reply:
This member is not really needed... you should make the listener to react on reception of the event, not at the ```execute```